bin/cli.js

```diff
@@ -76,11 +81,21 @@ require('yargs')
             args: argv.puppeteerArgs ? argv.puppeteerArgs.split(' ') : [],
             executablePath: argv.puppeteerExecutablePath,
         };
+
+        if (debug) {
+          launchArgs.headless = false;
+        }
```

Comment:
I feel it's easier to parse if we include this in the `launchArgs` definition above, eg. `headless: !debug`.